nixos/modules/programs/x2goserver.nix

```diff
@@ -0,0 +1,154 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.x2goserver;
+
+  x2goServerConf = pkgs.writeText "x2goserver.conf" ''
```

Comment:
Could you either provide a `config` or `extraConfig` option (`config` as described in [RFC42](https://github.com/NixOS/rfcs/pull/42)).

Reply:
config via generators.toINI added, that is really nifty.